scripts/tag-release.sh

```diff
@@ -37,6 +37,10 @@ find "$ROOT/install-manifests" \
      -type f -exec sed -i -E \
      's#(hasura/graphql-engine:)v(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(\-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?(\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?(.*)*$#\1'"${TAG}"'\9#' {} \;
 
+# add the latest tag to the catalog_versions file
+[ -n "$(tail -c1 catalog_versions)" ] && echo >> catalog_versions 
+echo $TAG $(cat catalog_version) >> catalog_versions
```

Comment:
I think you want to use `$ROOT` to build the paths here. (Also, the first line seems needlessly defensive to me, but I guess it’s fine if you want to keep it.)

Reply:
If someone modifies the file in an editor, it's not unlikely that it would add an extra newline. If we're turning that into an error, I think we should be careful not to add an extra one.

Reply:
I don’t understand—this code *adds* an extra newline if one isn’t already there. But your comment talks about being careful not to add an extra one, so I’m confused.

Reply:
I can either not add one at all, in which case I might end up missing a necessary newline character, or I can add one always, in which case I might end up with two.

Reply:
Ah, I see, sure. (I think it’d be okay to assume the newline will always be there, but you’re right, being defensive seems harmless.)